app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center,
+                 clean_power,
+                 average_rating,
+                 count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        suma = sum([self.calculate_washing_price(car) for car in cars
+                    if car.clean_mark < self.clean_power])
+        for car in cars:
+            if car.clean_mark <= self.clean_power:
```

Comment:
`serve_cars `- method, that takes a list of `Car`'s, washes only cars with `clean_mark `< `clean_power `of wash station and returns income of CarWashStation for serving this list of Car's, rounded to 1 decimal